requirements/base.txt

```diff
@@ -1,4 +1,4 @@
--e ./jmbase
--e ./jmbitcoin
--e ./jmclient
--e ./jmdaemon
```

Comment:
The `./` breaks the snapcraft python plugin for some reason.

Reply:
changes you made should be fine as long as `pip` can find the packages.